### PR TITLE
A0-1208: Stop asking for sudo in case of rpc

### PR DESCRIPTION
**Pull request by Yithis**
# Description

Stop asking for sudo in case of rpc "finalize" in cliain

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)

# Checklist:

- I have made corresponding changes to the existing documentation

### CODE REVIEW

File: bin/cliain/src/main.rs [hunk 1]
```diff
@@ -54,7 +55,14 @@ fn main() {
         command,
     } = Config::parse();
 
-    let seed = read_secret(seed, "Provide seed for the signer account:");
+    let seed = match command {
+        Command::Finalize {
```
Comment (maciejnems) [inline bin/cliain/src/main.rs:59]: I guess RotateKeys and GetStorage also do not need the seed phrase, right? And also SeedToSS58 should take a seed as an argument to command not as a signer seed. Could we handle those in a similar manner?

Comment (Yithis) [reply bin/cliain/src/main.rs:59]: Done

**Review by maciejnems — APPROVED** [review]

File: bin/cliain/src/main.rs (hunk 1 above)
Comment (DamianStraszak) [reply bin/cliain/src/main.rs:59]: Could you also move this piece of code to a separate function. `main` is already packed...

**Review by DamianStraszak — APPROVED** [review]

File: bin/cliain/src/main.rs (hunk 1 above)
Comment (Yithis) [reply bin/cliain/src/main.rs]: Done

**Pull request merged**